app/Http/Controllers/Client/ClientController.php

```diff
@@ -25,4 +28,21 @@ public function index()
             'latestProducts'
         ));
     }
+
+    public function register()
+    {
+        return view('layouts.client.auth.register');
+    }
+
+    public function postRegister(RegisterRequest $request)
```

Comment:
```suggestion
    public function register(RegisterRequest $request)
```
em đặt lại tên function nhé.